apps/api/src/app/widgets/widgets.controller.ts

```diff
@@ -158,6 +162,24 @@ export class WidgetsController {
     return await this.getSubscriberPreferenceUsecase.execute(command);
   }
 
+  @UseGuards(AuthGuard('subscriberJwt'))
+  @Post('/subscriber-preference')
```

Comment:
```suggestion
  @Patch('/preferences/:templateId')
```

What do you think about this?

Reply:
@scopsy Already updated.
however, I name a "Put" method and the reason was I did not see we use "Patch" in the api at me moment. 
Do you think we will use "Patch" more now?

Reply:
Patch is more appropriagte since it patches values and not replacing the entire object. I think semanticly it's better, wdyt? your call on this one

Reply:
I totally agree just was not sure regarding our conversions on the matter.